scripts/create_vxc_container_payload.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+set -e
+
+# Copyright (c) 2023 Wind River Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Validate number of arguments
+if [ $# -lt 4 ] || [ $# -gt 5 ]; then
+    echo "Usage: $0 <SDK_DIR> <OCI_IMG_DIR> <VXC_ARCH> <TOOL> [PACKAGE_PATH]"
+    echo "   eg. $0 /opt/wrsdk/wrsdk-vxworks7-raspberrypi4b-1.2 helloworld.build arm64 llvm"
```

Comment:
`e.g.,`
as described in the README..md the SDK patch is `~/Downloads/wrsdk` and the package name is `my_package.build`
Is it not better to use default values for this example?
`SDK_DIR=${1:-"~/Downloads/wrsdk"}`
OCI_IMG_DIR=${2:-"./my_package.build"}`
``